Helper/Cart.php

```diff
@@ -1032,9 +1032,10 @@ function ($item) use ($imageBlock, &$totalAmount, &$diff, $storeId) {
                 if(isset($item_options['attributes_info'])){
                     $properties = [];
                     foreach($item_options['attributes_info'] as $attribute_info){
+                        $attributeValue = is_string($attribute_info['value']) ? $attribute_info['value'] : var_export($attribute_info['value'], true);
```

Comment:
could we add some tests for this. Also why does `var_export` fix this issue? if this was an array var this would srtill break right?

would love to see tests + examples for this

Reply:
Agree with @roopakv here. If `$attribute_info['value']` is JSON, then wouldn't converting that JSON value into a string break something later on down the line as well?